simplivity/resources/datastores.py

```diff
@@ -176,3 +176,23 @@ def resize(self, size, timeout=-1):
         self.data = datastore_obj.data
 
         return self
+
+    def set_policy(self, policy_id, timeout=-1):
+        """Sets the backup policy for a datastore.
+
+        Args:
+            policy_id: The unique identifier (UID) of the policy that you want to associate with the datastore
+            timeout: Time out for the request in seconds.
+
+        Returns:
+            object: Datastore object.
+
+        """
+        resource_uri = "{}/{}/set_policy".format(URL, self.data["id"])
+        data = {"policy_id": policy_id}
+        out = self._client.do_post(resource_uri, data, timeout, None)
+        datastore = Datastores(self._connection)
+        datastore_obj = datastore.get_by_id(out[0]["object_id"])
+        self.data = datastore_obj.data
```

Comment:
should we just create a self.__refresh() method to update the data of the existing object?

Reply:
Created the __refresh() method